data/2016/sampleData/defaultconfig.yaml

```diff
@@ -129,6 +129,8 @@ decay:
 
   vehicleDamageTicks: 3 # damage vehicles every hour
   vacantFoundationTicks: 3 # destroy empty foundations after 1 hour
+  griefFoundationTimer: 259200000 # destroy grief foundations after 3 days
```

Comment:
could be in hours instead of ms, users don't need that level of precision

Reply:
#1859